admin/class-acc_user_importer-admin.php

```diff
@@ -606,6 +610,38 @@ private function proccess_expiry () {
 
 		$this->log_dual("Active members=$num_active, inactive members=$num_inactive");
 
+		// If option is set and there were membership changes, send email notification
+		// There is no checking done to ensure the notification email addresses are valid.
+		if (!empty($options['accUM_notification_emails']) &&
+			(!empty($new_users) || !empty($expired_users))) {
+			$email_addrs = $options['accUM_notification_emails'];
+
+			$title = "ACC membership change notification";
+			if (isset($options['accUM_notification_title'])) {
```

Comment:
Is there a setting for the title?

Reply:
Oops, looks like I forgot to add that option. Half-baked idea.  I will add when I have a chance on Monday.

Reply:
Added the code for the new option.